quickget

```diff
@@ -872,6 +892,23 @@ function get_openbsd() {
     make_vm_config "${ISO}"
 }
 
+function get_void() {
+    local FLAVOR=""
+    local HASH=""
+    local ISO=""
+    local URL=""
+    local arch="x86_64"
+    # TODO:  only one hardcoded version of void added.. Add more and make it automatic
+    if
```

Comment:
`if` what ?  - has no test or `fi` - this will not work - please set to draft until it at least passes a simple smoke test:
```
./quickget void
./quickget: line 910: syntax error near unexpected token `}'
./quickget: line 910: `}'
```